assets/scss/partials/components/_wrapper.scss

```diff
@@ -13,12 +13,23 @@
   &__main {
     width: 100%;
 
+    &--sidebarhidden {
+      margin-top: 20px;
+    }
+
     @include desktop_and_print {
       width: $content-width;
+      &--sidebarhidden {
+        margin-top: 0px;
+        width: 100%;
+      }
     }
 
     @include widescreen {
       width: calc(#{$content-ratio} * 80%);
+      &--sidebarhidden {
```

Comment:
Please always add whitespace between declaration groups such as BEM components as it was cherished in #292. It's not (just) for my OCD but for maintenance. If we'd stack wildly, it would be a mess to read.